frontend/pages/hosts/details/DeviceUserPage/DeviceUserPage.tsx

```diff
@@ -668,10 +668,13 @@ const DeviceUserPage = ({
         )}
         {!!host && showOSSettingsModal && (
           <OSSettingsModal
-            canResendProfiles={false}
-            hostId={host.id}
+            canResendProfiles={host.platform === "darwin"}
             platform={host.platform}
             hostMDMData={host.mdm}
+            resendRequest={(profileUUID: string) =>
```

Comment:
wondering if we should pull this out into it's own function to match other API calls